website/server/docsList.js

```diff
@@ -59,7 +59,9 @@ const apis = [
   '../Libraries/ReactNative/AppRegistry.js',
   '../Libraries/AppState/AppState.js',
   '../Libraries/Storage/AsyncStorage.js',
-  '../Libraries/Utilities/BackAndroid.android.js',
+  '../Libraries/Utilities/BackAndroid.js',
+  '../Libraries/Utilities/BackHandler.ios.js',
+  '../Libraries/Utilities/BackHandler.android.js',
```

Comment:
Does this cause there to be 3 separate pages? Lets consolidate it so that all the `BackHandler` docs are on one page.